modules/renter/hostdb/consts.go

```diff
@@ -9,11 +9,11 @@ import (
 const (
 	// maxHostDowntime specifies the maximum amount of time that a host is
 	// allowed to be offline while still being in the hostdb.
-	maxHostDowntime = 30 * 24 * time.Hour
+	maxHostDowntime = 10 * 24 * time.Hour
```

Comment:
this will help clear out the hostdb faster. If a host has been offline for that long, they need to re-announce to get found by renters.